aries-site/src/pages/templates/dashboards.mdx

```diff
@@ -72,4 +72,14 @@ Templates for 3 column and 4 column dashboards are coming soon.
   github="https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/templates/dashboards/TwoColumnDashboard"
 >
   <TwoColumnDashboard />
+</Example>
+
+### Three Column Dashboard
+
+<Example
+  screenContainer={{ scale: 0.5 }}
```

Comment:
Do we want to add the code here?

Reply:
oh ignore, we can add at the end once more is finalized.

Reply:
☝️ that was my thinking as well